target/linux/ath79/base-files/etc/board.d/02_network

```diff
@@ -343,6 +344,10 @@ ath79_setup_macs()
 		base_mac=$(mtd_get_mac_binary u-boot 130048)
 		wan_mac=$(macaddr_add "$base_mac" 1)
 		;;
+	trendnet,tew-823dru)
+		lan_mac=$(mtd_get_mac_text mac 4)
+		wan_mac=$(mtd_get_mac_text mac 24)
+		;;
```

Comment:
@ynezz Just found that this can be merged with `dlink,dir-825-c1|\` above, if you still want to change it.
Otherwise, I'll send a patch later.

Reply:
On a second thought: You might as well leave it. If I receive the label-mac, I will have to split it back anyway.

Reply:
Both MAC addresses are printed on the label but the WAN MAC address has a /FCC designation after it so I assume that maybe the official label MAC address.